lib/config.mk.in

```diff
@@ -5,6 +5,6 @@ CUDA_OBJS := @CUDA_OBJS@
 DEFINE_MACROS := @DEFINE_MACROS@
 prefix := @prefix@
 exec_prefix := @exec_prefix@
-CFLAGS := @MKCFLAGS@ -I@includedir@
+CFLAGS := @CPPFLAGS@ @MKCFLAGS@ -I@includedir@
```

Comment:
Yes, but it is CPPFLAGS, and this should be CFLAGS.

Will:

    CFLAGS="-I/opt/local/include" LDFLAGS="-L/opt/local/lib" ./configure

work for you?